CalculateTests.Tests/ProgramTests.cs

```diff
@@ -0,0 +1,47 @@
+using Calculate;
+
+namespace CalculateTests.Tests;
+
+    public class ProgramTests
+    {
+
+    //XUnit tests - Assert.Equal are generic by default
+       [Fact]
+        public void WriteLine_InvokesWriteLine_Success()
+        {
+            // Arrange
+            var expectedOutput = "Test Output";
+            string? actualOutput = null;
+            var program = new Program

```

Comment:
```suggestion
            Program program = new Program

```
this is what i did i think its better over a var

Reply:
✔